src/cargo/util/context/target.rs

```diff
@@ -135,7 +135,7 @@ fn load_config_table(gctx: &GlobalContext, prefix: &str) -> CargoResult<TargetCo
 fn parse_links_overrides(
     target_key: &ConfigKey,
     links: HashMap<String, CV>,
-    gctx: &GlobalContext,
+    _gctx: &GlobalContext,
```

Comment:
If this is unused, should we just remove it?

Reply:
For some reason I though this was a public API, but it's not. Removed.